README.md

```diff
@@ -7,7 +7,103 @@
 [chat-badge]: https://img.shields.io/discord/1082332781146800168.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2&style=flat-square
 [chat-url]: https://discord.gg/kKV8ZxyzY4
 
+# 🏁🏁 Build Status 🏁🏁
+You can view the current build status for the different packages [here](https://prefix-dev.github.io/reproducible-builds/).
 
+## Introduction
+This project aims to see if we can create reproducible builds for a variety of software packages.
+The project is based on definitions of the [Reproducible Builds](https://reproducible-builds.org/) project.
+It uses The [Rattler Build](https://github.com/prefix-dev/rattler-build) project, to build [Conda](https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html) packages.
+This repository has CI setup to build packages for the latests: windows, linux and osx platforms.
+It then tries to vary the build environment to see if the build is reproducible.
 
-## Build Status
-You can view the current build status [here](https://prefix-dev.github.io/reproducible-builds/).
+### Platform variations
+The following variations are tested (per platform):
+* **Linux**: Locale, timezone
+* **OSX**: Locale, timezone
+* **Windows**: None yet (TODO). Open to suggestions!
+
+This is similar to what other projects in the [Reproducible Builds](https://reproducible-builds.org/) project do, but for Conda packages.
+
+## Getting Started 🚀
+
+This project uses [pixi](https://github.com/prefix-dev/pixi) for project management.
+
+### Prerequisite
+Complete the following steps only once:
+
+1. Install Pixi by following the instructions on https://prefix.dev/
+2. Clone the repository and navigate to the root directory:
+    ```bash
+    git clone https://github.com/prefix-dev/reproducible-builds # or ssh
+    cd reproducible-builds
+    ```
+Ensure that all subsequent commands are executed in the project's root directory. When running the commands, the necessary environment will be set up automatically.
+
+## Quick Start
+To see if the project is setup correctly, run the following command:
+
+```bash
+# Build/Rebuild the boltons package
+pixi r reproduce boltons
+```
+
+### Configuration
+#### Config.yaml (what recipes to build) ⚙️
+The `config.yaml` file contains the configuration for the project. You can add `remote` or `local` recipes, that are local to the project or remote repositories.
+The config also contains the rattler-build version that its using, this way we can depend on unreleased versions. This project automatically sets this up for you.
+
+### SQLite Database (stores the build information) 📕
+This project uses a Sqlite database to store the build information. The database is created automatically when the project is setup.
+When running locally a local version of the database is created, this will ensure that you have a clean database to work with.
+You can also use the `--in-memory-sql` flag to use an in-memory database, which is useful for testing.
```

Comment:
let's add an example:
`pixi run repror --in-memory-sql build-recipe boltons` because sometimes for me is confusing these global options